client/components/email-verification/email-verification-dialog/index.jsx

```diff
@@ -56,13 +56,17 @@ class VerifyEmailDialog extends Component {
 
 	render() {
 		const strings = {
-			confirmHeading: this.props.translate( 'Please confirm your email address' ),
+			confirmHeading: this.props.translate( 'Please verify your email address:' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Please Verify Your Email Address' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/edit-gravatar-on-desktop)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).